NineChronicles.Headless.Executable/Commands/ReplayCommand.cs

```diff
@@ -240,9 +184,332 @@ public void Help([FromService] ICoconaHelpMessageBuilder helpMessageBuilder)
             }
             finally
             {
-                outputSw?.Dispose();
-                outputFs?.Dispose();
+                foreach (var disposable in disposables)
+                {
+                    disposable?.Dispose();
+                }
+
+                disposables.Clear();
             }
         }
+
+        [Command(Description = "Evaluate blocks and check state root hash")]
+        public int Blocks(
+            [Option('s', Description = "An absolute path of block storage.(rocksdb)")]
+            string storePath,
+            [Option('i', Description = "Target start block height. Tip as default. (Min: 1)")]
+            long startIndex = 1,
+            [Option('e', Description = "Target end block height. Tip as default. (Min: 1)" +
+                                       "If not set, same as START-INDEX.")]
```

Comment:
`START-INDEX` cannot be found in another options. I think this is the `-i` option, but other users seem to be confused.